docs/changelog.md

```diff
@@ -14,6 +14,8 @@ sidebar_label: Changelog
 - Adapted compilation flag so that the memory allocator uses JEMALLOC while counting allocated memory. [#401](https://github.com/memgraph/memgraph/pull/401)
 - `toString` function is now able to accept `Date`, `LocalTime`, `LocalDateTime` and `Duration` data types. [#429](https://github.com/memgraph/memgraph/pull/429)
 - Aggregation functions now return the openCypher-compliant results on `null` input and display the correct behavior when grouped with other operators. [#448](https://github.com/memgraph/memgraph/pull/448)
+- Correct inconsistencies and incorrect behavior with regards to sync replicas.
+#NoCommit Todo add link to documentation of replication
```

Comment:
Did you want to leave a comment on the changelog that wouldn't be visible?
Then, start with <! -- and end with -->

Reply:
Typically this is my way to write my "TODO" and I resolve all of them before setting the PR to "OPEN"
I am resolving the comment as I have already removed it.